src/ripple/protocol/impl/BuildInfo.cpp

```diff
@@ -36,12 +36,14 @@ namespace BuildInfo {
 char const* const versionString = "1.10.0-rc3"
 // clang-format on
 
-#if defined(DEBUG) || defined(SANITIZER)
+#if defined(DEBUG) || defined(SANITIZER) || defined(DEVELOP)
     "+"
 #ifdef GIT_COMMIT_HASH
     GIT_COMMIT_HASH
+#if ! defined(DEVELOP)
```

Comment:
Hmm, this logic is wrong in at least one way.
`SANITIZER` defined and this will crash rippled at runtime because of the version string not matching that semver version check